presto-docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -0,0 +1,140 @@
+========
+Pushdown
+========
+
+Presto can push down the processing of queries, or parts of queries, into the
+connected data source. This means that a specific function, or other operation,
+is passed through to the underlying database or storage system for processing.
+
+The results of this pushdown can include the following benefits:
+
+* improved overall query performance
+* reduced network traffic between Presto and the data source
+* reduced load on the remote data source
+
+Support for pushdown is specific to each connector and the relevant underlying
+database or storage system.
+
+Analysis and Confirmation
+-------------------------
+
+Pushdown depends on a number of factors:
+
+* generic support for pushdown for that function in Presto
+* function or operation specific support for pushdown in the connector
+* query that allows the detection of the function to push down
+* function needs to exist in the underlying system so it can process the
+  pushdown
```

Comment:
The connector needs to "implement" the function, but it can do it with a combination of expressions.
For example, avg can be pushed down (simplifying) as sum/count. 

We can remove this bullet as "function or operation specific support for pushdown in the connector" above covers it.